### ISSUE TITLE
Unhandled Rejection 

### ISSUE THREAD

**Issue by T-Nagui** [SupremeTechnopriest/react-idle-timer#183]
hello, i have this problem : 
Unhandled Rejection (InvalidStateError): Failed to execute 'postMessage' on 'BroadcastChannel': Channel is closed
---- 
use case : code is implimented only in authonticated Layout and when the user disconnect is other layout is charged 
i think the brodcast is still active and whay is the event is trigred.

any solution for this

sorry for my english ;p

**Comment by silenzium** [comment]
For me it is still not working with `next`, but @pcasa's solution does.
Wanted to checkout the documentation for crossTab but got a 404 there https://idletimer.dev/docs/props#crosstab

**Comment by SupremeTechnopriest** [comment]
Hey @silenzium the link is https://idletimer.dev/docs/api/props#crosstab

Can you `rm -rf node_modules package-lock.json && npm i` just to confirm you have the latest version?  Im definitely catching this error.  Maybe your package is stuck.  If it still persists can you describe the event in which this happens?  Is it after a HMR reload?

**Comment by SupremeTechnopriest** [comment]
Yeah @pcasa's solution doesn;t apply to the new api.  It sounds like you still have v4 installed.

**Comment by silenzium** [comment]
Ah ok there is no more object configuration.
And yeah, a forced version update helped, thank you!
